### PR TITLE
Hide fork menu item for read user

### PR DESCRIPTION
**Pull request by siddharthkp**
Tiny polish thing that came up in user testing. Want a review on implementation, not sure if this is a good way.

Read users don't have the ability to fork. Clicking this menu option leads to an error notification, so hiding it.

Note: Left the option in the command palette if you seek it out - it will still give you an error notification. Not sure if there's a way to show it but disable.

### CODE REVIEW

**Comment by lbogdan** [comment]
Build for latest commit https://github.com/codesandbox/codesandbox-client/pull/5279/commits/37aa511ce57ee707bda171f1789c3c86b32b01f5 is at https://pr5279.build.csb.dev/s/new.

**Review by SaraVieira — APPROVED** [review]
I would still wait for a review from christian

File: packages/app/src/app/overmind/effects/vscode/Workbench.ts
Comment (christianalfoni) [inline packages/app/src/app/overmind/effects/vscode/Workbench.ts:209]: The implementation makes sense, but I would revert the check. I typically do this trick:

```js
!(mySpecificScenarioWhereItShouldNotWork)

// so

!(this.controller.getState().editor?.currentSandbox?.teamId ===
        this.controller.getState().activeTeam &&
      this.controller.getState().activeWorkspaceAuthorization === 'READ')
```

It makes it more readable I think. Allows you to say: "this is the specific exception and I want it to work in all other scenarios" 😄

**Review by christianalfoni — CHANGES_REQUESTED** [review]
Looks good and it needs to be implemented here, but just a tiny comment. Just a preference thing, so no biggie 😄

File: packages/app/src/app/overmind/effects/vscode/Workbench.ts
Comment (siddharthkp) [reply packages/app/src/app/overmind/effects/vscode/Workbench.ts]: Made the changes :)

**Comment by siddharthkp** [comment]
Aw I want to revisit this soon

**Pull request closed without merging**
